code/game/objects/items/storage/pouch.dm

```diff
@@ -802,6 +802,28 @@
 	for(var/i = 1 to storage_slots)
 		new /obj/item/reagent_container/syringe(src)
 
+/obj/item/storage/pouch/engikit
+	name = "engineer kit pouch"
+	storage_flags = STORAGE_FLAGS_POUCH
+	icon_state = "construction"
+	desc = "It's specifically made to hold engineering items. Requires engineering skills to use effectively."
+	storage_slots = 7
```

Comment:
```suggestion
	storage_slots = 6
```

Reply:
See medkit pouch, It has the same number of slot as the medkit as well

Reply:
This isn't trying to be a medkit pouch. Also medical items are generally needed in lrger quantity, and physically are smaller.